roles/vald/vars/main.yml

```diff
@@ -3,9 +3,10 @@
 
 # Env file name
 vald_environment_secrets_file_name: "{{ service_identifier }}.secrets.env"
+vald_environment_wasmvm_file_name: "axelar.wasmvm.env"
 
 # binary names
 vald_default_binary_name: axelard
 
-vald_tofnd_host:
+vald_tofnd_host: "localhost"
 vald_from_address: broadcaster
```

Comment:
pease rename it to vald_broadcaster_address

Reply:
which one ?